.github/actions/e2e/setup-cluster/action.yaml

```diff
@@ -211,6 +211,11 @@ runs:
       else
         eksctl ${cmd} cluster -f clusterconfig.yaml
       fi
+
+      # Adding taints after all necessary pods have scheduled to the manged node group nodes
+      # amazon-cloudwatch-observability pods do no not tolerate CriticalAddonsOnly=true:NoSchedule and 
+      # EKS addons does not allow to add tolerations to the addon pods as part of the advanced configuration 
+      kubectl taint nodes CriticalAddonsOnly=true:NoSchedule --all
```

Comment:
Rather than banking on cluster creation and then tainting immediately after the fact, can this just be part of the ClusterConfig and then anything that doesn't tolerate it by default out-of-the-box, we just add additional addon configuration to?

Reply:
Currently, observability addon configuration does not support adding tolerations